### PR TITLE
Add ability to enable/disable from toolbar

### PR DESCRIPTION
**Pull request by thewheat**
Addresses https://github.com/igrigorik/videospeed/issues/464 by adding the ability to enable/disable VSC from the toolbar popup

Hope this works but happy to take on any feedback 👍

**Default view or when VSC is enabled**
image

**On enabling/disabling**
Will show a messaging asking to reload 
image

**If disabled it will show a "Enabled" button instead**
image

**Also added to options/settings page**
image

### CODE REVIEW

**Comment by jacobcolbert** [comment]
Looks good. I'll try it out.

**Comment by jacobcolbert** [comment]
We should make it so that the icon is grayed out if the extension is disabled. Needs to be clear what the current state is. Other than that it is working great for me.

**Comment by thewheat** [comment]
Totally makes sense! Just added that functionality! 🙌

**Disabled state**
image

**Comment by jacobcolbert** [comment]
Cool! Tagging @igrigorik to take a look. Thank you for taking on this feature. It was definitely needed.

File: popup.js [hunk 1]
```diff
@@ -10,4 +10,49 @@ document.addEventListener('DOMContentLoaded', function () {
...
+      "path": {
+        "19": "icons/icon19" + suffix,
+        "38": "icons/icon38" + suffix,
+        "48": "icons/icon48" + suffix
+      }
+    });
+  }
+
+  function settingsSavedReloadMessage(){
+    setStatusMessage("Saved. Reload page to see changes");
```
Comment (igrigorik) [inline popup.js]: Can we make this dynamic? 

- "Disabled. Reload to see changes."
- "Enabled. Reload to see changes."

**Comment by igrigorik** [comment]
Apologies about the slow turnaround. Overall, this looks great — great work @thewheat! Minor nitpick / request above before we land this.

**Comment by thewheat** [comment]
No worries @igrigorik! I'll see if I can get on to them this weekend 😃Just glad to contribute back to such a fantastic extension that has saved me countless hours if not days!

File: popup.js (hunk 1 above)
Comment (thewheat) [reply popup.js]: Done! 🙌

**Comment by igrigorik** [comment]
Awesome work, thank you!

**Pull request merged**